### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Yoyozilla**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+
+If applicable, add screenshots to help explain your problem.
+
+## Your Setup (please complete the following information)
+
+- Unity Version [e.g. 2018.3]
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/bug-report.md]:
```suggestion
- Unity Version [e.g. 2018.3.11f1]
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- MRTK Version [e.g. v2.0]
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/bug-report.md]: v2.0.0

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: ''
+assignees: ''
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Arbitrary bug filers going to know much about assignees? i.e. can we delete this line?

Comment (wiwei) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Or wait is this just auto-populated by editing the issue template.

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation Issue
+about: 'Something in our doc is missing or incorrect '
+title: ''
+labels: Documentation
+assignees: ''
+
+---
+
+**Describe the Issue**
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/documentation-issue.md]:
```suggestion
## Describe the Issue
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+A clear and concise what the issue
+
+**Feature Area**
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/documentation-issue.md]:
```suggestion
## Feature Area
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+What's incorrect? What's missing?
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/documentation-issue.md]: Maybe provide an example of a feature area (i.e. input system, how to guides, profile system)

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+**Existing Doc Link**
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/documentation-issue.md]:
```suggestion
## Existing Doc Link
```

File: .github/ISSUE_TEMPLATE/documentation-issue.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+If this is about something in an existing document, please provide link 
+
+**Additional context**
```
Comment (wiwei) [inline .github/ISSUE_TEMPLATE/documentation-issue.md]:
```suggestion
## Additional context
```

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 3 above)
Comment (Yoyozilla) [reply .github/ISSUE_TEMPLATE/bug-report.md]: yes

**Comment by wiwei** [comment]
Per feedback from chatting with @keveleigh offline, what are your thoughts on getting rid of the RC1 template? AFAICT the RC1 label is dead right now so they're basically the same thing and somewhat confusing for folks (which one do I choose if they do the same thing)

**Pull request merged**